shim.go

```diff
@@ -41,7 +41,7 @@ const (
 	KataShimType ShimType = "kataShim"
 )
 
-var waitForShimTimeout = 5.0
+var waitForShimTimeout = 10.0
```

Comment:
I was wondering if this is configurable via the API and therefore optionally through the runtime toml config for instance. Just a thought.